Readme.md

```diff
@@ -182,6 +182,9 @@ await cache.delete("key")
 await cache.delete_match("pattern:*")
 async for key in cache.keys_match("pattern:*"):
     ...
+async for key, value in cache.get_match("pattern:*"):
```

Comment:
I think you should add the `count` param for an example.

Reply:
Also, you have to add `get_match` to this line in Readme:
```
** Warning ** `cache.keys_match` does not work with this storage (works only if shards are disabled)
```

Reply:
thanks,

by the way do you mind if I rename `count` to `batch_size` ?

Reply:
Yup. Makes sense.

Reply:
Thank you so much